h/templates/client/thread.html

```diff
@@ -18,7 +17,7 @@
 </article>
 
 <!-- Reply count -->
-<div class="thread-reply" ng-show="vm.showReplyToggle(count('message')) && (!threadFilter.active() || count('message') == count('match'))">
+<div class="thread-reply" ng-show="!(vm.collapsed && !vm.isRoot) && count('message') > 0 && (!threadFilter.active() || count('message') == count('match'))">
```

Comment:
This condition is getting silly. It was already pretty bad but now I'm afraid I can barely read it. Can some of this move into the directive so that it's possible to give the various subconditions names?